src/pyfmi/fmil3_import.pxd

```diff
@@ -55,17 +61,59 @@ cdef extern from 'fmilib.h':
         fmi3_base_type_str     = 14,
         fmi3_base_type_enum    = 15
 
+    cdef enum fmi3_causality_enu_t:
+        fmi3_causality_enu_structural_parameter = 0,
+        fmi3_causality_enu_parameter            = 1,
+        fmi3_causality_enu_calculated_parameter = 2,
+        fmi3_causality_enu_input                = 3,
+        fmi3_causality_enu_output               = 4,
+        fmi3_causality_enu_local                = 5,
+        fmi3_causality_enu_independent          = 6,
+        fmi3_causality_enu_unknown              = 7
+
+    cdef enum fmi3_variability_enu_t:
+        fmi3_variability_enu_constant    = 0,
+        fmi3_variability_enu_fixed       = 1,
+        fmi3_variability_enu_tunable     = 2,
+        fmi3_variability_enu_discrete    = 3,
+        fmi3_variability_enu_continuous  = 4,
+        fmi3_variability_enu_unknown     = 5
+
+    cdef enum fmi3_initial_enu_t:
+        fmi3_initial_enu_exact,
```

Comment:
Should we also explicitly state the values here?

Reply:
Fixed. I am just worried though assigning `unknown` to a value and it also says "have to be last". For now I think it is fine, but for general enumerations I would be more careful. Because doesnt it mean that the value could change if more members are introduced in-between? Or am I wrong here?
Anyway, added values, see latest commit.